node/src/node/isometrics.hpp

```diff
@@ -0,0 +1,278 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+
+#include <blackhole/logger.hpp>
+#include <blackhole/scope/holder.hpp>
+
+#include <cocaine/format.hpp>
+#include <cocaine/context.hpp>
+#include <cocaine/logging.hpp>
+#include <cocaine/repository.hpp>
+
+#include "cocaine/service/node/manifest.hpp"
+#include "cocaine/service/node/profile.hpp"
+
+#include "cocaine/api/isolate.hpp"
+
+#include <metrics/factory.hpp>
+#include <metrics/registry.hpp>
+
+#include "node/pool_observer.hpp"
+
+#include "engine.hpp"
+
+namespace cocaine {
+namespace detail {
+namespace service {
+namespace node {
+
+namespace conf {
+    constexpr auto METRICS_POLL_INTERVAL_S = 2u;
```

Comment:
looks like we define constants in a similar to varible way - lower case with underscores